ons_alpha/bulletins/models.py

```diff
@@ -58,3 +60,13 @@ class BulletinPage(BasePage):
             ObjectList(BasePage.settings_panels, heading="Settings", classname="settings"),
         ]
     )
+
+    def get_latest_release(self):
+        parent = self.get_parent().specific
+        latest = BulletinPage.objects.live().descendant_of(parent).order_by('-release_date').first()
+        return latest
+
+    def get_previous_releases(self):
+        parent = self.get_parent().specific
+        previous = BulletinPage.objects.live().descendant_of(parent).exclude(id=self.id).order_by('-release_date')
```

Comment:
Better to use child of:

```suggestion
        parent = self.get_parent()
        previous = BulletinPage.objects.live().child_of(parent).exclude(id=self.id).order_by('-release_date')
```

ref: https://docs.wagtail.org/en/stable/reference/pages/queryset_reference.html#wagtail.query.PageQuerySet.child_of

Reply:
however, this should move to `BulletinSeriesPage` so this would become:

```suggestion
        previous = BulletinPage.objects.live().child_of(self).order_by('-release_date')
```